coralproject/talk
# [CORL-856] Show expert badge on comments

## What does this PR do?

<img width="387" alt="image" src="https://user-images.githubusercontent.com/5751504/74041334-b3d6db00-4982-11ea-835b-599675117c7d.png">

## How do I test this PR?

- Sign into the stream as admin/mod
- Configure > Convert to Q&A
- Apply
- Add your current user as an expert using the search/add expert control
- Go make a comment
- See that you have an expert badge
